packages/babel-helper-transform-fixture-test-runner/src/index.ts

```diff
@@ -474,7 +474,7 @@ function normalizeOutput(
 ) {
   const dir = path.resolve(
     path.dirname(fileURLToPath(import.meta.url)),
-    "../../../",
+    "../../..",
```

Comment:
Node.js 23 will now keep the trailing slash in the returned result, while previous node.js versions discarded it.

Reply:
https://github.com/nodejs/node/pull/55414
They seem to be trying to revert it.

Reply:
I was checking why citgm didn't catch this regression in Node.js, and I notice that we are for some reason failing during the install step: https://ci.nodejs.org/job/citgm-smoker/3489/nodes=alpine-latest-x64/testReport/junit/(root)/citgm/_babel_core_v7_25_8/

Reply:
Probably related to the path changes.